src/content/1.7/development/internationalization/make-a-new-language-available-to-be-used-in-the-back-office.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Make a new language available to be used in the Back Office
+aliases:
+  - /1.7/development/make_a_new_language_available_to_be_used_in_the_back_office
+  - /1.7/development/make-a-new-language-available-to-be-used-in-the-back-office
+---
+
+# Make a new language available to be used in the Back Office
+
+Before adding a language in Prestashop, you must check if the language is available in [Crowdin](https://crowdin.com/project/prestashop-official).
+If the language is not available in [Crowdin](https://crowdin.com/project/prestashop-official), you have to contact [Louise Bonnard on Crowdin](https://crowdin.com/profile/LouiseBonnard) for opening a project in the concerned language. 
```

Comment:
It is not relevant to write this information at this point because if there is no project on Crowdin for the language a contributor wants to activate, it means that there are no translations to be used in PrestaShop yet. If a translation project has a low completion level on Crowdin, unless its translators are very active, it is unecessary to send its translations to the software. In short, we consider adding a new supported language in PrestaShop when it has a 50% completion level at least.

Reply:
_Most languages are available in [Crowdin](https://crowdin.com/project/prestashop-official) for translation, but not all are available in PrestaShop (International > Translations > Add a language). Indeed, because of a low completion level on Crowdin, we do not send the translations to PrestaShop. However, it can smart to add a language if its translators are very active and want to see the results of their Crowdin translations quickly in their own shop._

Reply:
:ok_hand: